### DISCUSSION TITLE
ScriptReference/RectTransformUtility.ScreenPointToLocalPointInRectangle

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#265] [Notes]
# ScriptReference/RectTransformUtility.ScreenPointToLocalPointInRectangle

https://docs.unity3d.com/ScriptReference/RectTransformUtility.ScreenPointToLocalPointInRectangle.html

**Comment by radiatoryang** [comment]
Really important: Note the "cam = null" usage note in the docs!  
Also: this function won't actually give you the coordinates in anchored / UI pivot space... to convert:  
`localPoint += rect.rect.max;`  
(see [here](https://answers.unity.com/questions/1081112/convert-screen-point-to-ui-rectangle-coordinates.html) for info)
